Formula/mqttx-cli.rb

```diff
@@ -0,0 +1,12 @@
+class MqttxCli < Formula
+  desc "Open source MQTT 5.0 CLI client and MQTT X on the command-line"
+  homepage "https://github.com/emqx/MQTTX/tree/main/cli"
```

Comment:
The default page should be good enough
```suggestion
  homepage "https://github.com/emqx/MQTTX"
```